go/fuzz_test.go

```diff
@@ -122,3 +155,49 @@ func FuzzCombineProofs(f *testing.F) {
 		_, _ = CombineProofs(proofs)
 	})
 }
+
+type verifyJSON struct {
+	Proof *CommitmentProof
+	Ref   *RefData
+	Spec  *ProofSpec
+}
+
+func FuzzVerifyMembership(f *testing.F) {
```

Comment:
Just a code organisation nit: I would move the type and the test a bit up just after `FuzzVerifyNonMembership`.

Reply:
Done, I've moved it up. Thanks @crodriguezvega